package.json

```diff
@@ -3,6 +3,7 @@
   "license": "AGPL-3.0",
   "version": "0.1.0",
   "dependencies": {
+    "@codimd/markdown-it-task-lists": "git+https://git@github.com/truh/markdown-it-task-lists.git",
```

Comment:
Shouldn't this point to `git+https://github.com/codimd/markdown-it-task-lists.git`?

Reply:
I will update this once I'm done with the markdown-it-task-lists rewrite and it has been moved to the codimd project.

Reply:
I published the package as `@truh/markdown-it-task-lists` to make sure things work as intended. Feel free to either publish `@codimd/markdown-it-task-lists` or use `@truh/markdown-it-task-lists`, whatever you prefer.

Reply:
I would prefer ``@codimd/markdown-it-task-lists`` to have more control over the package for the future.